offload_vio/device_tx/vio_input.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+
+package vio_input_proto;
+
+message Vec3 {
+  double x = 1;
+  double y = 2;
+  double z = 3;
+}
+
+message IMUCamData
+{
+  fixed64 timestamp = 1;
```

Comment:
So its a bit odd since the timestamp inside of ILLIXR is a ulonglong of nanoseconds where as the timestamp output by VIO for the last integration time is a double since we convert it to seconds. Ill change the timestamp in the slowpose datatype to be fixed64 but i think last_cam_int_time should remain a double

Reply:
Yeah, I noticed this discrepancy too last night. Hopefully `RelativeClock` will fix all this once and for all.